app/models/comment/after_create_callback.rb

```diff
@@ -73,6 +74,11 @@ def create_watch(comment)
     @watch.save!
   end
 
+  def create_checker_id(comment)
+    product = comment.commentable
+    product.checker_id.blank? ? product.checker_id = comment.sender.id : return
```

Comment:
```suggestion
    product.checker_id = comment.sender.id if product.checker_id.blank?
```

三項演算子を使うと（returnも書いてあると）、返り値を重視してるように見えるので、このメソッド名からするとcreateするのが主題だと思うのでこちらの方がいいかもです〜。

Reply:
ありがとうございます！確かにこちらの方がわかりやすいですね。
修正しました！